packages/cli/src/commands/transfer.ts

```diff
@@ -0,0 +1,61 @@
+import { ZWeb3 } from 'zos-lib';
+import transfer from '../scripts/transfer';
+import { promptIfNeeded, networksList, InquirerQuestions } from '../prompts/prompt';
+import ConfigVariablesInitializer from '../models/initializer/ConfigVariablesInitializer';
+
+const name: string = 'transfer';
+const signature: string = name;
+const description: string = 'send funds in ether to the specified address';
+
+const register: (program: any) => any = (program) => program
+  .command(signature, undefined, { noHelp: true })
+  .usage('[options]')
+  .description(description)
+  .option('--units [units]', 'specify units.')
```

Comment:
```suggestion
  .option('--units <units>', 'specify units.')
```

I'd also consider renaming to `unit` instead of `units`, and indicate which values are valid ('eth', 'wei', etc).